app/views/public_recipe/index.html.erb

```diff
@@ -0,0 +1,20 @@
+    <%= stylesheet_link_tag "food", "data-turbo-track": "reload" %>
+        <%= stylesheet_link_tag "recipe", "data-turbo-track": "reload" %>
+<h1>Public Recipes</h1>
+<div class='no-recipes'>
+  <%= "No Public recipes available." if @public_recipes.empty? %>
+</div>
+<% @public_recipes.each do |recipe| %>
+    <ul class="recipe-list">
+        <li>
+            <div class="left">
+              <h2><%= recipe.name %></2>
+              <p>By: <%= recipe.user.name %></p>
+            </div>
+            <div class="right">
+               <p>Total food items: <%= recipe.recipe_foods.count %></p>
+               <p>Total price: <%= calculate_total_price(recipe) %></p>
+            </div>
+       </li>
+    </ul>
+<% end %>
```

Comment:
[REQUIRED] When a user clicks on any of the recipe items it should link to the recipe details page but I can't see any links here. Kindly fulfil this requirement.

[OPTIONAL] I think the `ul` should wrap the`<% @public_recipes.each do |recipe| %>` because I believe you want to create multiple lists but this will create multiple ul tags instead.